src/components/Editor/Editor.js

```diff
@@ -28,7 +33,7 @@ export default function Editor() {
       </div>
       <div className="form-control">
         <label>Alignment</label>
-        <div className="radio-group">
+        <div className="radio-group" onChange={(e) => setAlign(e.target.value)}>
```

Comment:
Nice use of [event bubbling](https://javascript.info/bubbling-and-capturing#bubbling)!